jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/SliderRange.java

```diff
@@ -30,42 +30,44 @@ private void setAttributes(int thumbIndex, int value, String newThumbStyle) {
   }
 
   private void reflectionSetValue(int thumbIndex, String newThumbStyle,
-                                  String newStyle, String trackStyle,int value, int left, int width) {
+                                  String newStyle, int value, int Left, int width) {
 
-    if (isSwitch(trackStyle, thumbIndex, value, left, width)) {
-      int right = left + width;
+    int localwidth = width;
+    int localLeft = Left;
+    if (isSwitch(thumbIndex, value, localLeft, width)) {
```

Comment:
Shouldn't be here used localWidth variable? The same for line 38, 70

Reply:
int right = localLeft + localWidth;

Reply:
I mean that we use "localWidth" variable in body of the method instead "width". Not everywhere is updated